.gitignore

```diff
@@ -1,3 +1,5 @@
 package.json
 package-lock.json
 node_modules/
+.ipynb_checkpoints/
+
```

Comment:
should we add all notebooks (*.ipynb) to the gitignore, to make sure local changes are not committed by mistake? In this way, we add a further check on the `git add`?